scripts/pythondistdeps.py

```diff
@@ -11,6 +11,10 @@
 # RPM python dependency generator, using .egg-info/.egg-link/.dist-info data
 #
 
+# Please know:
+# - Notes from an attempted rewrite from pkg_resources to importlib.metadata in
+#   2020 can be found in the message of the commit that added this line.
```

Comment:
😢

Reply:
Possible meanings of the emoji:

 - @Conan-Kudo is sad about the conclusion, but respects the way it was figured out.
 - The way this information is presented is making @Conan-Kudo sad.
 - @Conan-Kudo doesn't agree with the arguments presented in that commit message.
 - @Conan-Kudo is sad about current world situation and presented it here by accident.

I hope it is the first listed.

Reply:
It's definitely the first. 🤣

Reply:
Implemented to support both ways, see the latest fixup commit. And I've added tests for these cases:

    --provides --majorver-provides-versions 3.9 --majorver-provides-versions 2.7
    --provides --majorver-provides-versions 3.9,2.7
    --provides --majorver-provides-versions 3.9,2.7 --majorver-provides-versions 3.10
    --provides --majorver-provides-versions 3.9

Reply:
@Conan-Kudo @hroncok WDYT?

Reply:
LGTM

Reply:
I see this by a wrong comment, but +1.

Reply:
> I see this by a wrong comment, but +1.

What do you mean by that?